enterprise/authentication-email-password-setup.md

```diff
@@ -0,0 +1,10 @@
+# Authentication with email and password
```

Comment:
sorry this was probably my mistake but I'd use plural passwords and then it should be "Authentication with Email Addresses and Passwords" and same in the filename.

Reply:
updated